ui/src/components/Inventories/InventoryFormModal.js

```diff
@@ -0,0 +1,177 @@
+                      </Field>
+                      
+                    </Grid>
+                    <Grid item xs={12} sm={12}>
+                      <Field
+                        custom={{ variant: 'outlined', fullWidth: true}}
+                        name='description'
+                        label='Description'
+                        component={TextField}
+                      />
+                    </Grid>
+                      <Grid item xs={12} sm={12} >
+                        <Field
+                          custom={{ variant: 'outlined', fullWidth: true, }}
+                          inputProps = {{inputMode: 'numeric'}}
+                          name='averagePrice'
+                          label='Average Price'
+                          type='number'
+                          defaultValue = "0"
+                          component={TextField}
+                        />
+                      </Grid>
+                      <Grid item xs={12} sm={12}>
+                        <Field
+                          custom={{ variant: 'outlined', fullWidth: true, }}
+                          inputProps = {{inputMode: 'numeric'}}
+                          name='amount'
+                          label='Amount'
+                          type='number'
+                          defaultValue = "0"
+                          component={TextField}
+                        />
+                      </Grid>
+                    <Grid item xs={12} sm={12}>
+                    
+
+                    <Field
+                    select
+                    required
+                    custom={{ variant: 'outlined', fullWidth: true, }}
+                    name='unitOfMeasurement'
+                    label='Unit of Measurement'
+                    component={TextField}
+                    >
+                      {measurementUnitKeys.map((option) => (
+                        <MenuItem key={MeasurementUnits[option].abbreviation} value={option}>
+                          {MeasurementUnits[option].name}
+                            </MenuItem>
+                          ))}
+                      
+                    </Field>
+                    
+                      
+                    
+                  </Grid>
+                  <Grid item xs={12} sm={12}>
+                      <Field
+                        custom={{ variant: 'outlined', fullWidth: true, }}
+                        type="date"
+                        name='bestBeforeDate'
+                        label='Best before date'
+                        defaultValue = {date.getFullYear()+'-'+ (date.getMonth()+1) +"-"+date.getDate()}
+                        component={TextField}
+                      />
+                    </Grid>
+                    <Grid item xs={12} sm={12}>
+                        <Field
+                          custom={{ variant: 'outlined', fullWidth: true, }}
+                          defaultValue = {false}
```

Comment:
Remove any `defaultValue` in all of the `<Field/>` components since they're overridden by `initialValues`.